man/ff-deps.1.md

```diff
@@ -0,0 +1,135 @@
+% FF-DEPS(1)
+% Daniel de Kok, Sebastian Pütz
+% Apr 6, 2019
+
+NAME
+====
+
+**ff-deps** -- train dependency-based word embeddings with subword representations
+
+SYNOPSIS
+========
+
+**ff-deps** [*options*] *corpus* *output*
+
+DESCRIPTION
+===========
+
+The **ff-deps** trains dependency based word embeddings (Levy and Goldberg, 
+2014) using data from a *corpus* in CONLLX format. The corpus contains 
+sentences seperated by empty lines. Each sentence needs to be annotated with
+a dependency parse. After training, the embeddings are written to 
```

Comment:
a dependency parse -> a dependency graph

(does not necessarily have to come from a parser ;))